tests/core/test_wprintf.cpp

```diff
@@ -26,6 +29,22 @@ void PrintWide ( const wchar_t * format, ... )
   va_end ( args );
 }
 
+void PrintBigWide ( const wchar_t * format, ... )
+{
+  wchar_t buffer[MAX_CHARS_BIG] = { 0 };
+  va_list args;
+  va_start ( args, format );
+  int ret = vswprintf ( buffer, MAX_CHARS_BIG-1, format, args );
+  va_end ( args );
+  if (ret >= MAX_CHARS_BIG) {
```

Comment:
Can this ever return more than  MAX_CHARS_BIG-1?  It seems not, in which case this condition can be removed?